### PR TITLE
Bug 1112583 - Add a test to verify that a received call notification is ...

### PR DESCRIPTION
**Pull request by viorelaioia-zz**
...deleted when the SMS is deleted

### CODE REVIEW

**Comment by try-server-hook** [comment]
[viorelaioia viorelaioia](https://github.com/viorelaioia) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b9ee15553159)

**Comment by viorelaioia** [comment]
http://jenkins1.qa.scl3.mozilla.com/view/UI/job/flame-kk.ui.adhoc/658/console

**Comment by viorelaioia** [comment]
http://jenkins1.qa.scl3.mozilla.com/view/UI/job/flame-kk.ui.adhoc/668/console

**Comment by try-server-hook** [comment]
[viorelaioia viorelaioia](https://github.com/viorelaioia) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5407053b7ba6)

**Comment by try-server-hook** [comment]
[viorelaioia viorelaioia](https://github.com/viorelaioia) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=62496e126a91)

**Comment by try-server-hook** [comment]
[viorelaioia viorelaioia](https://github.com/viorelaioia) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=bbc752728a86)

**Comment by viorelaioia** [comment]
http://jenkins1.qa.scl3.mozilla.com/view/UI/job/flame-kk.ui.adhoc/679/console

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_sms_notification_removed_when_sms_deleted.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+from marionette import Wait
```
Comment (bebef1987) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_sms_notification_removed_when_sms_deleted.py]: import cat

**Comment by try-server-hook** [comment]
[viorelaioia viorelaioia](https://github.com/viorelaioia) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9e5d46595cdc)

**Pull request merged**